graphql_compiler/global_utils.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2019-present Kensho Technologies, LLC.
+from collections import OrderedDict
+from itertools import chain
+
+import six
+
+
+def _get_dag_with_keys_in_reverse_topological_order(dag):
+    """Return an OrderedDict representing a DAG with keys ordered in reversed topological order.
```

Comment:
Might be useful to briefly explain what reversed topological order is. 

Probably also nice to mention that the output may not be unique.

Reply:
I think reversed topological order is fairly obvious, given that the reader knows what a topological sort is: the reversed order of a valid topological order. I think this is fine.
I do agree that we need to point out the toposort is not unique. We could just reword it to `ordered in a reversed topological order`, and it'll be correct.